scripts/west_commands/blobs.py

```diff
@@ -151,6 +153,23 @@ def fetch(self, args):
                 self.dbg('Blob {module}: {abspath} is up to date'.format(**blob))
                 continue
             self.inf('Fetching blob {module}: {abspath}'.format(**blob))
+
+            if blob['license-click-through']:
+                user_input = input("For this blob, need to read and accept license to continue."
+                                   "Read it? [y/n]: ")
+                if user_input.upper() != "Y":
+                    self.inf('Warn: skip fetching this blob.')
+                    continue
+
+                with open(blob['license-abspath']) as license_file:
+                    license_content = license_file.read()
+                    print(license_content)
+
+                user_input = input("Accept license to continue? [y/n]: ")
+                if user_input.upper() != "Y":
+                    self.inf('Warn: skip fetching this blob.')
```

Comment:
Printing a warning with `inf`?

```suggestion
                    self.wrn('Skipped fetching this blob.')
```

Reply:
Thanks. Will use it.